plugins/ssr-plugin.js

```diff
@@ -114,8 +114,8 @@ const SSRBodyTemplate = createPlugin/*:: <SSRBodyTemplateDepsType,SSRBodyTemplat
 
         for (let url of criticalChunkUrls) {
           const crossoriginAttr = url.startsWith(__webpack_public_path__)
```

Comment:
Now think that this condition might not be the correct thing to check against, since `__webpack_public_path__` could be a CDN. Perhaps it should be `url.startsWith("/")`, but I suppose that doesn't handle the case where the absolute url is same-origin (although this case might not matter).

Reply:
It looks like the [legacy ssr template](https://github.com/fusionjs/fusion-core/blob/master/src/plugins/ssr.js#L139) used `webpackPublicPath.startsWith('http')` which is similar to what you mentioned. What about checking  for`CDN_URL`?

```js
const crossoriginAttr = process.env.CDN_URL ? ' crossorigin="anonymous"' : '';
```